internal/service/kafka/kafka.go

```diff
@@ -25,6 +25,7 @@ func aivenKafkaSchema() map[string]*schema.Schema {
 	aivenKafkaSchema["default_acl"] = &schema.Schema{
 		Type:        schema.TypeBool,
 		Optional:    true,
+		ForceNew:    true,
```

Comment:
Why do we force recreation of the resource here? I think this filed only makes sense when during resource creation

Reply:
we cannot have this field edited, editing it should require a resource recreation

Reply:
Can we make it `Computed` after creation?  Or make it use `CreateOnlyDiffSuppressFunc`?